src/OpenTracingActiveRecordBehavior.php

```diff
@@ -0,0 +1,165 @@
+<?php
+
+namespace Websupport\OpenTracing;
+
+use CEvent;
+use CModelEvent;
+use OpenTracing\Scope;
+
+/**
+ * Class OpenTracingActiveRecordBehavior
+ * @package Websupport\OpenTracing
+ *
+ * @method \CActiveRecord getOwner()
+ */
+class OpenTracingActiveRecordBehavior extends \CActiveRecordBehavior
+{
+    /** @var string */
+    public $opentracing = 'opentracing';
+
+    /**
+     * Whether we should hook to before and after find methods.
+     * Be careful with this option. There is option, where you can disable calling afterFind method
+     * directly in the code.
+     * @var bool
+     */
+    public $traceFind = false;
+
+    /**
+     * Whether we should hook to before and after save methods.
+     * @var bool
+     */
+    public $traceSave = true;
+
+    /**
+     * Whether we should hook to before and after delete methods.
+     * @var bool
+     */
+    public $traceDelete = true;
+
+    /** @var Scope */
+    private $activeScope = null;
```

Comment:
`null` is default, define it in annotation if it's required

Reply:
done